css/style.css

```diff
@@ -301,13 +303,24 @@ h1, h2, h3, h4, h5, h6 {
     
 }
 #numberfact{
-    font-family: mainfont;
+    font-family: 'Berkshire Swash', cursive;
```

Comment:
The font you added, "Berkshire Swash", is not much user-friendly. Could you add another more user-friendly and readable font, like Roboto or Open Sans? Or you can even leave the font as it was previously.